statistics/selectivity.go

```diff
@@ -354,6 +354,20 @@ func (coll *HistColl) Selectivity(ctx sessionctx.Context, exprs []expression.Exp
 				if ok {
 					continue
 				}
+				// where {"0" / 0 / "false" / false / null} or A or B ... the '0' constant item should be ignored.
+				if c, ok := cond.(*expression.Constant); ok {
```

Comment:
According to our logic in `buildSelection` and `DeleteTrueExprs`, I think we should (1) avoid `DeferredExpr` and `ParamMarker` and (2) use `EvalBool()` or `ToBool()` to convert it to a bool.

Reply:
nice suggestion, updated